sdk/testutil/testlog.go

```diff
@@ -15,12 +17,31 @@ func init() {
 	sendTestLogsToStdout = os.Getenv("NOLOGBUFFER") == "1"
 }
 
-func TestLogger(t testing.TB) *log.Logger {
```

Comment:
We really shouldn't delete functions from this library as someone may be using them so the change would not be backwards compatible.

You can however leave the original function intact and indicate with a comment that it is deprecated:

```
// Deprecated: use Logger(t)
func TestLogger(t testing.TB) *log.Logger {
```
see: https://rakyll.org/deprecated/